.github/workflows/ci.yaml

```diff
@@ -114,6 +114,57 @@ jobs:
         run: sudo make build
       - name: test-e2e
         run: sudo make test-e2e-for-fluentd # containerd interaction requires sudo
+  e2e-tests-for-splunk:
+    strategy:
+      fail-fast: false
+      matrix:
+        go: [ '1.20', '1.21' ]
+        os: [ ubuntu-latest ] # TODO: Add Windows e2e tests: https://github.com/aws/shim-loggers-for-containerd/issues/68
```

Comment:
May not be easy as just defining a windows OS here to define the windows e2e test, as a lot of the subsequent steps will change.

Reply:
yes there should be a windows branch